### PR TITLE
[Merged by Bors] - feat: port LinearAlgebra.AffineSpace.Midpoint

### PR DESCRIPTION
**Pull request by mcdoll**
Added two new lemmas to make the `simpNF` linter happy.

---

- [x] depends on: #2692

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by semorrison** [comment]
This PR/issue depends on:
* leanprover-community/mathlib4#2692
By **[Dependent Issues](https://github.com/z0al/dependent-issues)** (🤖). Happy coding!

**Comment by semorrison** [comment]
This PR/issue depends on:
* ~~leanprover-community/mathlib4#2692~~
By **[Dependent Issues](https://github.com/z0al/dependent-issues)** (🤖). Happy coding!

File: Mathlib/LinearAlgebra/AffineSpace/Midpoint.lean [hunk 1]
```diff
@@ -18,43 +18,38 @@ import Mathlib.LinearAlgebra.AffineSpace.AffineEquiv
 
 * `midpoint R x y`: midpoint of the segment `[x, y]`. We define it for `x` and `y`
   in a module over a ring `R` with invertible `2`.
-* `add_monoid_hom.of_map_midpoint`: construct an `add_monoid_hom` given a map `f` such that
+* `AddMonoidHom.of_MapMidpoint`: construct an `AddMonoidHom` given a map `f` such that
```
Comment (eric-wieser) [inline Mathlib/LinearAlgebra/AffineSpace/Midpoint.lean:21]:
```suggestion
* `AddMonoidHom.ofMapMidpoint`: construct an `AddMonoidHom` given a map `f` such that
```

**Comment by ChrisHughes24** [comment]
bors merge

**Pull request closed without merging**